src/todos/todos.database.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable, NotFoundException } from '@nestjs/common'
+import { initialTodos } from 'mocks/todos'
+import { TodoItem } from 'types/todos'
+
+@Injectable()
+export class TodosDatabaseService {
+  static todos: TodoItem[] = initialTodos
```

Comment:
Array is not the best choice of data structure when you need random access by unique ID and your data is unordered. Let's use `Map` instead

Reply:
Fixed!